man/chpl.rst

```diff
@@ -545,11 +545,11 @@ OPTIONS
     Print the names and source locations of unused functions within the
     user program.
 
-**-s, --set <config param>[=<value>]**
+**-s, --set <config>[=<value>]**
 
-    Overrides the default value of a configuration parameter in the code.
-    For boolean configuration variables, the value can be omitted, causing
-    the default value to be toggled.
+    Overrides the default value of a configuration param, type, var,
+    or const in the code.  For boolean configs, the value can be
```

Comment:
Just like in the comment above, it currently sets any config to `true`, not just booleans.  If they're explicitly typed it will likely result in a type mismatch error, if not they'll turn into booleans.